src/shaka_scheme/system/base/String.hpp

```diff
@@ -18,12 +18,12 @@ class String {
       str(str) {}
 
   // make new string with length "size"
-  String(int size) :
-      str(' ', size) {}
+  String(std::size_t size) :

```

Comment:
Thanks for changing this to `std::size_t`. This code was sort of old so we needed this,